src/Particular.Analyzers/DiagnosticDescriptors.cs

```diff
@@ -171,5 +171,21 @@ public static class DiagnosticDescriptors
             category: "Code",
             defaultSeverity: DiagnosticSeverity.Warning,
             isEnabledByDefault: true);
+
+        public static readonly DiagnosticDescriptor DateTimeAssignedToDateTimeOffset = new DiagnosticDescriptor(
+            id: DiagnosticIds.DateTimeAssignedToDateTimeOffset,
+            title: "Prevent a DateTime from being implicitly assigned to a DateTimeOffset",
+            messageFormat: "Do not implicitly assign a DateTime value to a DateTimeOffset variable",
+            category: "Code",
+            defaultSeverity: DiagnosticSeverity.Warning,
+            isEnabledByDefault: true);
+
+        public static readonly DiagnosticDescriptor UseUtcNow = new DiagnosticDescriptor(
+            id: DiagnosticIds.UseUtcNow,
+            title: "Use DateTime.UtcNow or DateTimeOffset.UtcNow unless the value is being used for displaying the current date-time in a user's local time zone",
```

Comment:
Similar:

```suggestion
            title: "DateTime.UtcNow or DateTimeOffset.UtcNow should be used instead of DateTime.Now and DateTimeOffset.Now, unless the value is being used for displaying the current date-time in a user's local time zone",
```